src/needs-parens.js

```diff
@@ -38,8 +40,16 @@ function needsParens(path) {
           );
         case "bin":
           return parent.type === "**" && name === "left";
+        default:
+          return false;
+      }
+    case "new": {
+      if (isLookupNode(parent)) {
+        return true;
       }
-    // else fallthrough
+
+      return false;
```

Comment:
`return isLookupNode(parent)`

Reply:
@czosel i use `false` for future, we only start print parens better, code change very often, i refactor all code after complete main improvement :+1: